.github/workflows/publish.yml

```diff
@@ -233,7 +233,7 @@ jobs:
             docker manifest create ${t} --amend ${t}-amd64 --amend ${t}-arm64v8
             docker manifest create ${{ env.GHCR_IMAGE_NAME }}:latest --amend ${t}-amd64 --amend ${t}-arm64v8
           done
-        if: startsWith(github.ref, 'refs/tags/')
+        if: startsWith(github.ref, 'refs/tags/') && contains('-pre-', github.ref)
```

Comment:
This doesn't seem like it will work as intended. Perhaps the extra part needs a "not"

Reply:
You are correct. I shouldn't put up pull requests when tired. 😅